doubly_linked_list/doubly_linked_list.py

```diff
@@ -74,7 +109,23 @@ def move_to_end(self, node):
     return self.head.value
 
   def delete(self, node):
-    pass
+    if node.next is None and node.prev is None and self.head is node and self.tail is node:
```

Comment:
Just a thought: the first two conditions are given when head and tail are equal to node (assuming head and tail nodes were setup properly).